src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -541,6 +541,48 @@ p, li { white-space: pre-wrap; }
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QCheckBox" name="threadStateChangeCallstackCollectionCheckBox">
```

Comment:
Consider adding a TODO to add a tooltip to explain to the users what these new options mean

Reply:
I added two tooltips.